client/state/billing-transactions/test/actions.js

```diff
@@ -162,4 +170,71 @@ describe( 'actions', () => {
 			} );
 		} );
 	} );
+
+	describe( '#requestBillingTransaction', () => {
```

Comment:
Would be great if we could use jest for these tests. Not a requirement though, as previous tests seem to haven't been migrated yet.